internal/scaffold/olm-catalog/csv.go

```diff
@@ -234,6 +234,31 @@ func (s *CSV) setCSVDefaultFields(csv *olmapiv1alpha1.ClusterServiceVersion) {
 			{Type: olmapiv1alpha1.InstallModeTypeAllNamespaces, Supported: true},
 		}
 	}
+	if len(csv.Spec.Icon) == 0 {
+		csv.Spec.Icon = []olmapiv1alpha1.Icon{
+			olmapiv1alpha1.Icon{
+				Data:      "Placeholder data",
+				MediaType: "Placeholder mediaType"}}
+	}
+	if len(csv.Spec.Maintainers) == 0 {
+		csv.Spec.Maintainers = []olmapiv1alpha1.Maintainer{
+			olmapiv1alpha1.Maintainer{
+				Name:  "Placeholder name",
+				Email: "Placeholder email"}}
+	}
+	if len(csv.Spec.Keywords) == 0 {
+		csv.Spec.Keywords = []string{"Placeholder keyword"}
+	}
+	if csv.Spec.Provider == (olmapiv1alpha1.AppLink{}) {
+		csv.Spec.Provider = olmapiv1alpha1.AppLink{
+			Name: "Placeholder name",
+			URL:  "Placeholder URL"}
+	}
+
+	if len(csv.Spec.MinKubeVersion) == 0 {
```

Comment:
Is this actually mandatory? CSV's will deploy without this field, and will register a status saying no minikube version is found.

Reply:
Is possible to use it successfully when the status is no minikube version is found? Also, is not it mandatory to publish the project in the community operator as well?